packages/venia-concept/src/components/SearchBar/suggestions.js

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useCallback } from 'react';
+import { arrayOf, bool, func, shape, string } from 'prop-types';
+
+import { mergeClasses } from 'src/classify';
+import SuggestedCategories from './suggestedCategories';
+import SuggestedProducts from './suggestedProducts';
+import defaultClasses from './suggestions.css';
+
+const Suggestions = props => {
+    const { products, searchValue, setVisible, visible } = props;
+    const { filters, items } = products;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const onNavigate = useCallback(() => {
+        setVisible(false);
+    }, [setVisible]);
+
+    if (!visible || !filters || !items) {
```

Comment:
I think this condition is a bit to greedy. We probably just want to hide the related section if the list is empty for that thing. If no filters, just don't show the categories, section. If no items, don't show the suggested products.

Reply:
Perhaps true, but it does complicate the component a bit. Either both of those properties should either be present (once the query has run) or not present. They'll still render if the lists are empty.